src/main/deploy/Autos/OneBall

```diff
@@ -0,0 +1 @@
+TaxiOutFromFender.path
```

Comment:
Remove deploy/Autos/* from git tracking. Should already by in the .gitignore file